package.json

```diff
@@ -53,10 +54,14 @@
     "zod-gpt": "^0.12.1"
   },
   "devDependencies": {
+    "@eslint/js": "^9.6.0",
     "@jest/globals": "^29.7.0",
     "@types/debug": "^4.1.12",
     "@types/supertest": "^6.0.2",
     "@types/yargs": "^17.0.32",
+    "eslint": "^9.6.0",
+    "globals": "^15.8.0",
+    "husky": "^9.0.11",
```

Comment:
we added husky but don't have any pre-commit hooks?

Reply:
😅

Reply:
fixed now